data/campaigns/Sceptre_of_Fire/scenarios/8_The_Dragon.cfg

```diff
@@ -118,11 +118,42 @@
     [event]
         name=prestart
 
+        {VARIABLE found_forge no}
+
         [objectives]
             side=1
             [objective]
                 description= _ "Get all heroes to the end of the tunnel"
                 condition=win
+                [show_if]
+                    {VARIABLE_CONDITIONAL found_forge equals no}
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Move Thursagan to the forge"
+                condition=win
+                [show_if]
+                    {VARIABLE_CONDITIONAL found_forge equals yes}
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Defend Thursagan until the Sceptre of Fire is complete"
+                condition=win
+                [show_if]
+                    {VARIABLE_CONDITIONAL found_forge equals yes}
+                [/show_if]
+            [/objective]
```

Comment:
Has a behaviour change, it will be shown as well while Thursagan is in the forging. While that would fit, the [objective below serves already for that part.

Reply:
To match the old behaviour, it needs a second filter:
```
                [show_if]
                    {VARIABLE_CONDITIONAL found_forge boolean_equals yes}
                    [have_unit]
                        id=Thursagan
                    [/have_unit]
                [/show_if]
```